bindings/matrix-sdk-ffi/src/timeline/mod.rs

```diff
@@ -196,6 +196,13 @@ impl Timeline {
         })
     }
 
+    /// Mark the room as read by trying to add a read receipt to the latest
+    /// event.
+    pub async fn mark_as_read(&self, receipt_type: ReceiptType) -> Result<(), ClientError> {
+        self.inner.mark_as_read(receipt_type.into(), ReceiptThread::Unthreaded).await?;
```

Comment:
Does this still work if the latest event in the timeline is from a thread? 🤔

Reply:
You're right, I don't think it would, *if we supported thread receipts*. But we're not, at the moment, so the unthreaded receipt should be sufficient (the thread relation doesn't matter when we're not considering them, as they appear as responses ordered by the sync order).

Reply:
Can we document this?

Reply:
I've even removed the thread parameter from `Timeline::mark_as_read`, quoting the commit message here:

It doesn't make sense to set a thread identifier for the latest event: the event might not belong to that thread, and the SDK would need to check that, since the latest event isn't reachable from the outside world (the reason why `mark_as_read` had been introduced). So let's remove it for now, and add comments related to threaded receipts.